### PR TITLE
Add deprecation notices for old k8s

### PR DESCRIPTION
**Pull request by meyskens**
Adds upgrade and install notes for 1.2.0 not supporting old kubernetes

### CODE REVIEW

File: content/en/docs/installation/upgrading/upgrading-1.1-1.2.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+linkTitle: "v1.1 to v1.2"
+weight: 810
+type: "docs"
+---
+
+> The upgrade process for upgrading to `v1.2` is very Kubernetes version specific. Please check the version of your cluster using `kubectl version` and follow the steps required for your version of Kubernetes.
+
+### Kubernetes `1.15.x` and below
+
+cert-manager `v1.2` will no longer support Kubernetes `v1.15` and below following the Kubernetes upstream version support. We advise you to consider upgrading to Kubernetes 1.16 or above. If this isn't possible we advice to stay with the cert-manager `v1.1.x` releases.
```
Comment (wallrj) [inline content/en/docs/installation/upgrading/upgrading-1.1-1.2.md:12]:
```suggestion
cert-manager `v1.2` will no longer support Kubernetes `v1.15` and below following the Kubernetes upstream version support. We advise you to consider upgrading to Kubernetes 1.16 or above. If this isn't possible we advise you to stay with the cert-manager `v1.1.x` releases.
```

**Review by wallrj — CHANGES_REQUESTED** [review]
Just one small typo and there are a couple of spell check errors to be fixed.

**Review by wallrj — APPROVED** [review]
/lgtm

**Comment by meyskens** [comment]
This is merged into the wrong branch! It should be reverted!

**Pull request merged**